homeassistant/components/husqvarna_automower_ble/config_flow.py

```diff
@@ -16,8 +16,23 @@
 from homeassistant.components.bluetooth import BluetoothServiceInfo
 from homeassistant.config_entries import SOURCE_BLUETOOTH, ConfigFlow, ConfigFlowResult
 from homeassistant.const import CONF_ADDRESS, CONF_CLIENT_ID, CONF_PIN
+from homeassistant.helpers.device_registry import format_mac
 
-from .const import DOMAIN, LOGGER
+from .const import DOMAIN, LOGGER, MANUFACTURER
+
+BLUETOOTH_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_PIN): str,
+    }
+)
+
+USER_SCHEMA = BLUETOOTH_SCHEMA.extend(
+    {
+        vol.Required(CONF_ADDRESS): str,
+    }
+)
```

Comment:
This results in having the data flipped around (PIN first), is it acceptable or can it be avoided?